### PR TITLE
Add proteinfold-specific CRG config

### PR DESCRIPTION
**Pull request by athbaltzis**
---
name: proteinfold-specific CRG config
about: A new proteinfold-specific config for the sge cluster at CRG
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [X] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [X] Add your custom config file to the `conf/` directory
- [X] Add your documentation file to the `docs/` directory
- [X] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [X] Add your custom profile to the `README.md` file in the top-level directory
- [X] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -98,6 +98,7 @@ jobs:
           - "uzh"
           - "vai"
           - "vsc_ugent"
+          - "crg"
```
Comment (JoseEspinosa) [inline .github/workflows/main.yml:101]: If I am not wrong this list is alphabetically ordered, take a look

File: nfcore_custom.config [hunk 2]
```diff
@@ -30,6 +30,7 @@ profiles {
   cfc_dev      { includeConfig "${params.custom_config_base}/conf/cfc_dev.config" }
   cheaha       { includeConfig "${params.custom_config_base}/conf/cheaha.config" }
   computerome  { includeConfig "${params.custom_config_base}/conf/computerome.config" }
+  crg        { includeConfig "${params.custom_config_base}/conf/crg.config" }
```
Comment (JoseEspinosa) [inline nfcore_custom.config:33]:
```suggestion
  crg         { includeConfig "${params.custom_config_base}/conf/crg.config" }
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM! 
Just two small suggestion feel free to merge after they have been corrected  😄

File: .github/workflows/main.yml (hunk 1 above)
Comment (athbaltzis) [reply .github/workflows/main.yml]: Yes, you are right

**Pull request merged**